### ISSUE TITLE
"Peer certificate cannot be authenticated with given CA certificates." on Ubuntu 16.04

### ISSUE THREAD

**Issue by rehmanzile** [lastpass/lastpass-cli#242]
LastPass CLI v0.7.0 getting the following error on Ubuntu 16.04 for the ```lpass login``` command.

```shell
Error: Peer certificate cannot be authenticated with given CA certificates.
```

**Comment by bcopeland** [comment]
Already reported, this should be reported to ubuntu as we don't make the ubuntu packages.

**Comment by rehmanzile** [comment]
@bcopeland How can I pick up the changes if Ubuntu doesn't generate the debs? I tried compiling from the source and it did not work?

**Comment by bcopeland** [comment]
What happened when you compiled from source?

Backported fix for 0.7.0 tarball is available here: https://github.com/lastpass/lastpass-cli/releases/tag/v0.7.2.  If you have all the build dependencies, dpkg-buildpackage should work from there.

Yes, we are looking into a way to provide binaries through distro channels.  I'm personally not too keen on running a PPA but we are looking into it.
